src/frontends/paddle/CMakeLists.txt

```diff
@@ -2,8 +2,9 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 
-add_subdirectory(src)
+include_directories(${CMAKE_CURRENT_SOURCE_DIR}/../../core/dev_api)
 
+add_subdirectory(src)
```

Comment:
why need to modify this CMakeList?

Reply:
I wanted to access a utility func in the validation_util header file in the core_dev folder. I modified the file to tell MakeTool the location of the file.

Reply:
@Pey-crypto 
I see. Do you want to call `ov::util::normailize_axis` in the `openvino/core/validation_util.hpp`? If so, `ov::util::normailize_axis` is a very simple function, which is used to correct axis when it is negative. You can implement it like `(value < 0) ? value + max : value;`.
Need not to modify the Makefile and import more dependency.